### DISCUSSION TITLE
Works with 6850 LTE

### DISCUSSION THREAD

**Discussion by ikabod** [Freetz-NG/freetz-ng#916] [Allgemein]
Tested yesterday with the latest stable version of Freetz-NG. I used only the defaults and removed some packages (can have a look in the config file, if it is important).

The flash function works also (tools/push_firmware, if I'm correct).

I use the box with a SIM card from MEO (Portugal) which can be used for Internet data and SMS only (no phone calls are possible).

If you have any questions, please let me know.

Peter

**Comment by fda77** [comment]
Thank you for feedback!  Can you please attach a screenshot of the freetz-webinterface: System > AVM-Info
The CPU details could only be read from a device, not from a firmware image
1b1ed42f0

**Comment by ikabod** [comment]
Here you go:

![image](https://github.com/Freetz-NG/freetz-ng/assets/12196222/f87eb1cc-38ca-4bb5-b83f-c9316c5f4572)

**Reply by fda77** [reply]
Many thanks!
Is this a new device with 2 day uptime or does it forget its operating hours?

**Reply by ikabod** [reply]
Yes it is new, but I consider to send it back, since it is only possible to use 70 characters to send a SMS.

I used Freetz-NG to change this to at least 420 characters and sending of SMS longer than 70 characters is now working, but I got an error after sending in the log that the message could not be sent and this is true.

Today I opened a ticket at TP-Link and asked if they have a LTE-Router with SMS functionality. Ifs so, the FritzBox goes back. Can't believe the SMS function at all: Hidden under "Internet" -> "Zugangsdaten" (German), list ordered by the oldest message first, delete every SMS separately with a confirmation message. Sending needs to push a button on the router. There is some strange thinking at the AVM company....

**Reply by fda77** [reply]
AVM's lte devices a very expensive. You only want to send and receive SMS? I used smstool many years ago with a umts usb stick. I dont know if it still works and with "new" lte devices. 
https://freetz-ng.github.io/freetz-ng/make/smstools3.html

**Reply by ikabod** [reply]
Yes, the device was expensive, but I thought I invest in good hard- and software. Hardware may be ok (so far no problems), while the software is not. I do not want to send SMS only with it, but I need this function.

I tried smstools3 today, but the make command will not complete with it:

![VirtualBox_Freetz Ubuntu 23 04](https://github.com/Freetz-NG/freetz-ng/assets/12196222/1139e88f-297c-4b6f-97e8-6853e2a72aa8)

I removed the UMTS patch of the box as suggested on the homepage, but I guess the repository is too old in the meantime.

I do not waste more time with this. The Archer MR600 from TP-Link has SMS functionality and I will test it.

**Reply by fda77** [reply]
The new gcc you are using has a changed default. Update freetz and run `make smstools3-recompile`. As SMS use 2G services it could still run
c7a195bf339ac069f5e98a9624fb5bacfdf545f4

**Reply by ikabod** [reply]
Thanks, got it to work now, but it does not work out of the box:

Starting SMStools3 ... data directory not configured, failed. 

This message also appears, if I check "als Flash"...

I do not want to read a documentation for this - why is it not working with the internal storage?

PS: Unpatching the UMTS modul of the FritzBox will make this smstools3 to not work at all.

**Reply by fda77** [reply]
`¯\_(ツ)_/¯`
